src/features/sign-up/model/hooks/useRegistrationMutation.ts

```diff
@@ -49,7 +49,9 @@ export const useRegistrationMutation = (
       SessionModel.storeSession(session);
 
       AnalyticsService.login(user.id).then(() => {
-        AnalyticsService.track('Login Successful');
+        AnalyticsService.track('Signup Successful', {
+          'User ID': user.id,
+        });
```

Comment:
We should move 'User ID' from the payload.
Please add 
```
this.mixpanel.getPeople().set({
  'User ID': user.id,
})
``` 
to `login` method of `MixpanelAnalytics` class